### PR TITLE
Add saltboot redeployment sub chapter

### PR DESCRIPTION
**Pull request by aaannz**
# Description

Added saltboot redeployment guide as a terminal deployment subchapter.

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [X] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/21884

### CODE REVIEW

File: modules/retail/pages/saltboot-redeployment.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
+[[retail.deploy.force_redeployment]]
+= Force saltboot image redeployment
+
+Usually systems provisioned by saltboot are redeployed or repartitioned automatically when new image is available or saltboot partitioning changes.
+
+Occasionally however it is needed to force saltboot to redeploy image or repartition disk even when automation would not do so. For these situations saltboot offers three ways to force redeployment or repartitioning:
```
Comment (keichwa) [inline modules/retail/pages/saltboot-redeployment.adoc:6]:
```suggestion
Occasionally however it is needed to force saltboot to redeploy image or repartition disk even when automation would not do so.
For these situations saltboot offers three ways to force redeployment or repartitioning:
```

**Review by keichwa — APPROVED** [review]
Thanks, Ondrej!  Mostly just cosmetic suggestions.

When merged, the doc squad should probably walk thru again to propose more normalization changes according to our style guide.  For example, we always want to use the {salt} instead of literal salt or Salt to make sure that it is properly capitalized.  In the procedure, we probably can follow more Lana's style ideas.

**Review by 0rnela — APPROVED** [review]

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Added Saltboot redeployment sub chapter in the Retail Guide
```
Comment (0rnela) [inline .changelog:7]:
```suggestion
- Added Saltboot redeployment subchapter in the Retail Guide
```

File: modules/retail/pages/retail-deploy-terminals.adoc [hunk 3]
```diff
@@ -28,9 +27,6 @@ After you have registered new terminals, always check the {productname} {webui}
 The terminals must not have directly connected to the {productname} Server by mistake.
 ====
 
-
-
-
 == Create A Hardware Type Group
```
Comment (0rnela) [inline modules/retail/pages/retail-deploy-terminals.adoc:30]:
```suggestion
== Create a Hardware Type Group
```

File: modules/retail/pages/saltboot-redeployment.adoc [hunk 4]
```diff
@@ -0,0 +1,114 @@
+[[retail.deploy.force_redeployment]]
+= Force saltboot image redeployment
+
+Usually systems provisioned by saltboot are redeployed or repartitioned automatically when new image is available or saltboot partitioning changes.
```
Comment (0rnela) [inline modules/retail/pages/saltboot-redeployment.adoc:4]:
```suggestion
Systems provisioned by Saltboot are usually redeployed or repartitioned automatically when a new image is available or Saltboot partitioning changes.
```

File: modules/retail/pages/saltboot-redeployment.adoc [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+Occasionally however it is needed to force saltboot to redeploy image or repartition disk even when automation would not do so.
+For these situations saltboot offers three ways to force redeployment or repartitioning:
+
+* Force redeployment using salt grains
```
Comment (0rnela) [inline modules/retail/pages/saltboot-redeployment.adoc:9]:
```suggestion
* Force redeployment using {salt} grains
```

File: modules/retail/pages/saltboot-redeployment.adoc [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+* Force redeployment using salt pillars
+
+[WARNING]
+====
+Repartitioning of a terminal removes all data stored on the terminal hard disk, including any persistent partitions.
+====
+
+== Force saltboot redeployment using salt grains
+
+Saltboot redeployment grains have no side effects and does not require any further configuration.
```
Comment (0rnela) [inline modules/retail/pages/saltboot-redeployment.adoc:20]:
```suggestion
Saltboot redeployment grains have no side effects and do not require any further configuration.
```

File: modules/retail/pages/saltboot-redeployment.adoc [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+Limitation is that terminal must be accessible by [systeminfo]``salt``.
```
Comment (0rnela) [inline modules/retail/pages/saltboot-redeployment.adoc:21]:
```suggestion
The limitation is that terminal must be accessible by [systeminfo]``salt``.
```

File: modules/retail/pages/saltboot-redeployment.adoc [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+
+.Procedure: Forcing a Saltboot to Redeploy Image
+. On the {productname} Server, at the command prompt, as root, apply this Salt state:
```
Comment (0rnela) [inline modules/retail/pages/saltboot-redeployment.adoc:24]:
```suggestion
. On the {productname} Server, as root, apply this {salt} state at the command prompt:
```

**Pull request merged**